TombEngine/Objects/TR4/Trap/tr4_joby_spikes.cpp

```diff
@@ -1,49 +1,48 @@
 #include "framework.h"
-#include "tr4_joby_spikes.h"
-#include "Specific/level.h"
+#include "Objects/TR4/Trap/tr4_joby_spikes.h"
+
+#include "Game/animation.h"
 #include "Game/collision/collide_room.h"
 #include "Game/collision/Point.h"
 #include "Game/control/control.h"
-#include "Game/animation.h"
-#include "Sound/sound.h"
-#include "Game/Lara/lara.h"
 #include "Game/effects/effects.h"
 #include "Game/items.h"
+#include "Game/Lara/lara.h"
+#include "Sound/sound.h"
+#include "Specific/level.h"
 
 using namespace TEN::Collision::Point;
 
+// TODO: Need to test and adapt formula for scaling to other heights.
+
 namespace TEN::Entities::Traps
 {
-    constexpr auto JOBY_SPIKES_DAMAGE = 8;
+    constexpr auto JOBY_SPIKES_HARM_DAMAGE          = 8;
     constexpr auto JOBY_SPIKES_EXTRA_ROTATION_SPEED = 2;
-    constexpr auto JOBY_SPIKES_SCALE_INCREMENT = 3;
-    constexpr auto JOBY_SPIKES_MAX_SCALE = 3328; //3 blocks + 1 click
-    //TODO: Need to test and adapt formula for scaling to other heights.
+    constexpr auto JOBY_SPIKES_SCALE_INCREMENT      = 3;
+    constexpr auto JOBY_SPIKES_MAX_SCALE            = BLOCK(3.25f);
 
     void InitializeJobySpikes(short itemNumber)
     {
         auto& item = g_Level.Items[itemNumber];
 
         auto& angleRotationSpeed    = item.ItemFlags[0];
         auto& spikeLength           = item.ItemFlags[1];
-        auto& clockworkDirection    = item.ItemFlags[2];
+        auto& clockworkDirection    = item.ItemFlags[2]; // ??
```

Comment:
Is a boolean. When the trap is initialized, it randomly asign 0 or 1, to indicate if it will rotate to the right or to the left (clockwork direction or oposite).